Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Stage 0, "build-stage", based on Node.js, to build and compile the frontend
+FROM tiangolo/node-frontend:10 as build-stage

```

Comment:
I just noticed, that this build image seems rather abandoned.
Since we do not use the chrome/puppeteer stuff for testing (but cypress) i took the liberty to adapt your PR in my own fork.

I also just noticed that you used windows line endings and i converted them to unix (as the rest of the project)

Reply:
I preserved your contributions though, so you still get the credit ;-)

Would you accept closing this PR in favor of #156 ?

Reply:
@lb1c certainly :)